cmd/agentctl2/utils/db_utils.go

```diff
@@ -0,0 +1,654 @@
+
+package utils
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/gogo/protobuf/proto"
+
+	"github.com/ligato/cn-infra/db/keyval"
+	acl "github.com/ligato/vpp-agent/api/models/vpp/acl"
+	interfaces "github.com/ligato/vpp-agent/api/models/vpp/interfaces"
+	ipsec "github.com/ligato/vpp-agent/api/models/vpp/ipsec"
+	l2 "github.com/ligato/vpp-agent/api/models/vpp/l2"
+	l3 "github.com/ligato/vpp-agent/api/models/vpp/l3"
+	nat "github.com/ligato/vpp-agent/api/models/vpp/nat"
+
+	linterface "github.com/ligato/vpp-agent/api/models/linux/interfaces"
+	ll3 "github.com/ligato/vpp-agent/api/models/linux/l3"
+
+	"errors"
+)
+
+const (
+	ACLPath          = "config/vpp/acls/v2/acl/"
```

Comment:
These shouldn't be hard-coded.

See example how the aclplugin retrieves key prefix: https://github.com/ligato/vpp-agent/blob/731e9b4b9b2e37407e805221dcab9ccbd0482353/plugins/vpp/aclplugin/descriptor/acl.go#L66